internal/distribute/rest/distribute.go

```diff
@@ -53,6 +55,22 @@ type logAndVerifier struct {
 	sigV   note.Verifier
 }
 
+var (
+	doOnce                 sync.Once
+	counterDistRestAttempt monitoring.Counter
+	counterDistRestSuccess monitoring.Counter
+)
+
+// InitMetrics defines the metrics for the omniwitness and all dependencies. Must be called before
+// Main in order to have any effect. Only the first call to this method has any effect.
+func InitMetrics(mf monitoring.MetricFactory) {
+	doOnce.Do(func() {
+		const logIDLabel = "logid"
+		counterDistRestAttempt = mf.NewCounter("distribute_rest_attempt", "Number of attempts the RESTful distributor has made for the log ID", logIDLabel)
+		counterDistRestSuccess = mf.NewCounter("distribute_rest_success", "Number of times the RESTful distributor has succeeded for the log ID", logIDLabel)
```

Comment:
Do these need to be incremented below?

Reply:
Whoops, yeah. I did this but then managed to revert it somehow? I baffle myself sometimes.